rules-tests/DeadCode/Rector/ClassMethod/RemoveUselessReturnTagRector/Fixture/multiline_comment_for_core_php_return.php.inc

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\ClassMethod\RemoveUselessReturnTagRector\Fixture;
+
+class DemoFile
+{
+    /**
+     * @return UserlandClass
+     *   A userland class is fine.
```

Comment:
I remember drupal uses this, Rector is using phpdoc parser which iirc, recognize next comment of doc is in same line, next line is considered different scope from current doc that's why the `@return` got removed.

Reply:
This can be applied with enable `textBetweenTagsBelongsToDescription` flag on

https://github.com/rectorphp/rector-src/blob/625fcccc475c8a6b3bde49e5e64e66354ed4299a/packages/BetterPhpDocParser/PhpDocParser/BetterPhpDocParser.php#L59-L61

But that cause **another bug** like in https://github.com/rectorphp/rector-src/pull/4590#discussion_r1272236459 that need to be resolved in **phpdoc-parser** itself first.